.github/workflows/osx-sccache.yml

```diff
@@ -0,0 +1,52 @@
+name: osx-sccache
+
+on:
+  push:
+    branches: [master, release-*, gha-test-*]
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  max_threads: 16
+  pre_command: cd /opt/git/diem/
+
+jobs:
+  update-sccache:
```

Comment:
I didn't see where we upload the sccache artifacts to S3 here.  Is it implicitly handled in x?

Reply:
yes